records_mover/db/bigquery/load_job_config_options.py

```diff
@@ -324,13 +324,14 @@ def add_load_job_csv_config(unhandled_hints: Set[str],
     quiet_remove(unhandled_hints, 'datetimeformat')
 
     if hints.datetimeformattz in ['YYYY-MM-DD HH:MI:SSOF',
+                                  'YYYY-MM-DD HH24:MI:SSOF',
                                   'YYYY-MM-DD HH:MI:SS']:
         pass
     else:
         cant_handle_hint(fail_if_cant_handle_hint, 'datetimeformattz', hints)
     quiet_remove(unhandled_hints, 'datetimeformattz')
 
-    if hints.timeonlyformat == 'HH24:MI:SS':
+    if hints.timeonlyformat in ['HH24:MI:SS', 'HH:MI:SS']:
```

Comment:
`timeonlyformat` on the other hand does not list `HH:MI:SS` as a supported value - even though it's identical to `HH24:MI:SS `, which is.